.github/workflows/test-server.yaml

```
@@ -0,0 +1,22 @@
+name: Server unit test
+
+on:
+  push:
+  paths:
+    - 'server/**'
```

```suggestion
    paths:
      - 'server/**'
```
The syntax seems invalid